LibraryApp/src/app/components/Book/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { View, Text, Image } from 'react-native';
+
+import styles from './styles';
+import PropTypes from 'prop-types';
+
+function Book({ title, author, photo }) {
+  return (
+    <View style={styles.container} >
+      <Image
+        source={photo && {uri: photo} || require('../../assets/book_placeholder.png')}
```

Comment:
I would import `'../../assets/book_placeholder.png'` as a constant and then use it here.

I would use a ternary to decide which image you will show